examples/kubernetes/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG BASE_IMAGE_NAME=vault.habana.ai/gaudi-docker/1.16.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.2
+ARG BASE_IMAGE_TAG=latest
+
+FROM ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG} AS optimum-habana
+
+ARG GAUDI_SW_VER=1.16.1
```

Comment:
if you introduce this argument then you should use it every where! even on your base